File: java/src/main/java/com/oltruong/Problem6.java
package com.oltruong;

import java.util.stream.LongStream;

class Problem6 {

    private static final long NUMBER = 100;

    static long getResult() {
        long sumOfNumber = computeSumOfNumber();
        long sumSquared = sumOfNumber * sumOfNumber;

        long sumOfSquares = LongStream.rangeClosed(1, NUMBER).map(n -> n * n).sum();

        return sumSquared - sumOfSquares;
    }

    private static long computeSumOfNumber() {
        return NUMBER * (NUMBER + 1) / 2;
    }

}

File: python/main/problem6.py
def get_problem6():
    return find_difference_sum_squares_square_sum(100)


def find_difference_sum_squares_square_sum(number):
    return (number * (number + 1) / 2) * (number * (number + 1) / 2) - sum([i * i for i in range(1, number + 1)])
